### PR TITLE
[DE] Update `Client/Beatmap editor/Beat snap divisor`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Tasks

- [x] Add image for beat snap divisor
- [x] [Beat Snapping](https://osu.ppy.sh/wiki/de/Beatmapping/Beat_snapping): Beat-Positionierung
- [x] `dense rhythm`: Rhythmus mit höherer Intensität

## Notes

- more dense rhythm --> schnellerer Rhythmus (not sure on this one)
- I added an explanation for buzz sliders in parantheses

### Links

Sites, that are useful for this and surrounding topics:

- [Taktarten - Studyflix](https://studyflix.de/allgemeinwissen/taktarten-4031)
- [Taktart - osu!wiki](https://osu.ppy.sh/wiki/de/Music_theory/Time_signature)
- [Quantisierung](https://de.wikipedia.org/wiki/Quantisierung_(Musikbearbeitung)) --> see Beat Snapping

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by The-Last-Cookie** [comment]
`Beat-Orientierung` does not express well enough in my opinion that [beat snapping](https://osu.ppy.sh/wiki/en/Beatmapping/Beat_snapping) means to attach hit objects to ticks on the timeline.

While thinking about it, maybe `Beat-Positionierung` is a better alternative?

> Also would translate more dense rhythm to dichterer Rythmus as dichterer means more dense whereas schnellerer means faster.

I am aware of this issue. If I remember correctly, there is an article using a similar expression. I will try to investigate more into other possible translations.

**Comment by Y2nn1c** [comment]
I agree that Beat-Positionierung does get the point across in a better way. However, how about Beat-Ausrichtung? 
It would get the point better across that we are talking about an objects position being based on the beats since Beat-Ausrichtung sounds more like us talking about beats directly and how they are positioned instead of something being positioned based on them. 
But if you don't like Beat-Ausrichtung I suggest we take Beat-Positionierung. It's not a bad suggestion and I doubt we will find a translation without having to do a compromise in some way.
Also since I'm a native speaker, can I approve of this merge? I just got Github today and I'm honestly totally lost.

**Comment by The-Last-Cookie** [comment]
Thanks for your contribution!

Of course, if you agree with the changes being made to the article, you can approve the Pull Request (PR) after I have resolved the open tasks.

For your infomation, I got feedback via discord for this article. So, we now have the following candidates (This list might be expanded later on):

- ~~Beat-Platzierung~~ (already sorted out)
- Beat-Positionierung
- Beat-Ausrichtung
- Beat-Raster
- Beat-Rasterung

For now, I prefer the first two translations. As a side note, I planned to use `Distanzraster` for Distance Snap which might be to similar to `Beat-Raster`. By the way, you may want to take a look at my [wiki notes](https://github.com/The-Last-Cookie/osu-wiki-notes/tree/master/guidelines).

I want to give this a fresh mind tomorrow before taking the final decision.

File: wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md
Comment (Y2nn1c) [inline wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md:12]:
```suggestion
Der Taktteiler bestimmt die Anzahl von Beats in einem Takt auf der [Zeitleiste](/wiki/Client/Beatmap_editor/Timelines). Dieser Teiler drückt dies Form eines Bruchs aus. Ein geringere Bruch bedeutet mehr Beats pro Takt. Es gibt die Optionen
1/1, 1/2, 1/3, 1/4, 1/6 und 1/8 wobei 1/4 den Standart Taktteiler darstellt.
```

File: wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md
Comment (Y2nn1c) [inline wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md:16]:
```suggestion
Der Beatmap-Editor unterstützt für den Taktteiler die Optionen 1/1, 1/2, 1/3, 1/4, 1/5, 1/6, 1/7, 1/8, 1/9, 1/12 und 1/16
wobei 1/4 der Standartwert des Taktteilers darstellt.
```

File: wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md
Comment (Y2nn1c) [inline wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md:32]:
```suggestion
1/1 (ein ganzer Takt), 1/2 (ein halber Takt) und 1/4 (ein Vierteltakt) sind die am häufigsten verwendeten Notenwerte. 
Die Taktarten 1/3 (ein Tripeltakt) und 1/6 werden oft im Walzer verwenden und kommen daher nur sehr selten vor.
```

**Comment by Y2nn1c** [comment]
> Thanks for your contribution!
> 
> Of course, if you agree with the changes being made to the article, you can approve the Pull Request (PR) after I have resolved the open tasks.
> 
> For your infomation, I got feedback via discord for this article. So, we now have the following candidates (This list might be expanded later on):
> 
>     * ~Beat-Platzierung~ (already sorted out)
> 
>     * Beat-Positionierung
> 
>     * Beat-Ausrichtung
> 
>     * Beat-Raster
> 
>     * Beat-Rasterung
> 
> 
> For now, I prefer the first two translations. As a side note, I planned to use `Distanzraster` for Distance Snap which might be too similar to `Beat-Raster`. By the way, you may want to take a look at my [wiki notes](https://github.com/The-Last-Cookie/osu-wiki-notes/tree/master/guidelines).
> 
> I want to give this a fresh mind tomorrow before taking the final decision.

Alright, I'm also fine with either of the two mentioned ones. 
However I would use Objektdistanz for Distance Snap as Distanzraster may lead to an misunderstanding due to the editor having a grind and would imply some sort of connection to it and to the function Grid Snap.

File: wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md
Comment (Y2nn1c) [reply wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md:32]:
```suggestion
1/1 (ein ganzer Takt), 1/2 (ein halber Takt) und 1/4 (ein Vierteltakt) sind die am häufigsten verwendeten Notenwerte, da viele Songs mit Rhythmen höherer Intensität komponiert werden. Taktarten wie 1/3 (ein Tripeltakt) und 1/6 werden oft im Walzer verwendet, wobei ein Takt in 3 oder 6 gleiche Teile aufgeteilt wird.
```
Would change it to this so as to not repeat the word mit twice.
I'm fine with the rest of the changes, if we are done with this one would you be done on your end?

**Comment by Y2nn1c** [comment]
Oh also what do did you decide on from our list of suggestions?

File: wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md
Comment (The-Last-Cookie) [inline wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md:10]:
```suggestion
Der **Taktteiler** (auch *Beat Snap Divisor* genannt) ist eine Einstellung, die bestimmt, wie der Editor [Beats](/wiki/Music_theory/Beat) darstellt. Dieser Prozess nennt sich [Beat-Positionierung](/wiki/Beatmapping/Beat_snapping). Der Taktteiler befindet sich in der rechten oberen Ecke des Editors.
```

File: wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md
Comment (The-Last-Cookie) [inline wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md:12]:
```suggestion
Durch den Taktteiler wird die Feinheit der [Hit-Objekte](/wiki/Gameplay/Hit_object) auf der [Zeitleiste](/wiki/Client/Beatmap_editor/Timelines) bestimmt. Der Teiler drückt in Form eines Bruchs aus, in wie viele Teile ein Beat aufgeteilt werden soll. Ein höherer Notenwert bedeutet, dass mehr Noten im selben Takt platziert werden können, und umgekehrt.
```

**Comment by The-Last-Cookie** [comment]
> Oh also what do did you decide on from our list of suggestions?

Both `Positionierung` and `Ausrichtung` are equally good, so I'm going with my personal preference which is `Positionierung`.

**Review by Y2nn1c — APPROVED** [review]
Checked everything and agree with all the changes.
Everything is gramatically correct as well and follows the given guidelines and professional standart that osu! wishes to uphoald. 
I'm from germany so this Pull Request is completely finished and ready to go live.

**Comment by The-Last-Cookie** [comment]
Thanks a lot for your help!

File: wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md
Comment (FreezyLemon) [inline wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md:10]: Seems sensible to mention "beat snapping" since we also mention "Beat Snap Divisor".

```suggestion
Der **Taktteiler** (auch *Beat Snap Divisor* genannt) ist eine Einstellung, die bestimmt, wie der Editor [Beats](/wiki/Music_theory/Beat) darstellt. Dieser Prozess nennt sich [Beat-Positionierung](/wiki/Beatmapping/Beat_snapping) (engl. **beat snapping**). Der Taktteiler befindet sich in der rechten oberen Ecke des Editors.
```

File: wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md
Comment (FreezyLemon) [inline wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md:16]:
```suggestion
Der Beatmap-Editor unterstützt für den Taktteiler elf verschiedene Einstellungen im Bereich von 1/1 bis 1/16.
```

File: wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md
Comment (FreezyLemon) [inline wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md:34]:
```suggestion
Die anderen Einstellungen des Taktteilers sind ungewöhnlich und sollten mit Bedacht eingesetzt werden: Solange ein Song oder ein Abschnitt darin nicht speziell mit ungewöhnlichen Notenlängen komponiert wurde, sind seltene Taktarten wie 1/5 or 1/16 normalerweise ein Hinweis darauf, dass die Beatmap falsch [getimt](/wiki/Beatmapping/Timing) wurde. 1/16 wird jedoch üblicherweise für Buzz-Slider (kurze, sich viele Male wiederholende Slider) verwendet.
```

**Review by FreezyLemon — APPROVED** [review]
Good stuff :+1:

"Taktteiler" works really well. Just have some soft suggestions

**Comment by The-Last-Cookie** [comment]
*Note: The review by Yannic2112 was removed by GitHub (his account is [likely temporarily] banned)*

**Review by TicClick — APPROVED** [review]

**Pull request merged**
